pom.xml

```diff
@@ -614,6 +616,81 @@
       </modules>
     </profile>
 
+    <profile>
+      <id>alpine</id>
+      <activation>
+        <property>
+          <name>release</name>
+          <value>true</value>
+        </property>
+      </activation>
+
+      <distributionManagement>
+        <downloadUrl>https://maven.pkg.github.com/aklivity/zilla/</downloadUrl>
+        <repository>
+          <id>github</id>
+          <url>https://maven.pkg.github.com/aklivity/zilla/</url>
+        </repository>
+        <snapshotRepository>
+          <id>github</id>
+          <url>https://maven.pkg.github.com/aklivity/zilla/</url>
+        </snapshotRepository>
+      </distributionManagement>
+
+      <properties>
+        <docker.context>alpine</docker.context>
+        <docker.platforms>linux/x86_64</docker.platforms>
+        <docker.tagSuffix>-alpine</docker.tagSuffix>
+        <docker.tagAlt>alpine</docker.tagAlt>
```

Comment:
The alpine version is the only one to need a `tagSuffix` because it will share the same semver. When we want to start pushing the incubator build the `tagSuffix` can be set on build using a unique id like the commit hash or something.